src/sharding/Shard.js

```diff
@@ -99,29 +90,80 @@ class Shard extends EventEmitter {
     this._exitListener = this._handleExit.bind(this, undefined);
   }
 
+  /**
+   * Process of the shard (if {@link ShardingManager#mode} is `process`)
+   * @type {?ChildProcess}
+   * @readonly
+   */
+  get process() {
+    return childProcess;
+  }
+
+  /**
+   * Worker of the shard (if {@link ShardingManager#mode} is `worker`)
+   * @type {?Worker}
+   * @readonly
+   */
+  get worker() {
+    return workerThread;
+  }
+
+  /**
+   * Parent port of the worker threads (if {@link ShardingManager#mode} is `worker`)
+   * @type {?MessagePort}
+   * @readonly
+   */
+  get parentPort() {
+    return Worker.parentPort ?? null;
```

Comment:
Since this class is on the master side, isn't the parentPort always gonna be null?